src/GeneralStateTestsFiller/stTransactionTest/CreateTransactionEOF1Filler.yml

```diff
@@ -139,3 +141,42 @@ CreateTransactionEOF1:
          balance: 30000
        6295ee1b4f6dd65047762f924ecd367c17eabf8f:
          shouldnotexist: 1
+
+    - indexes:
+       data: ':label deploying_valid_eof1'
+       gas: !!int -1
+       value: !!int -1
+      network:
+        - 'Berlin'
+      result:
+       a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+         nonce: 1
+       6295ee1b4f6dd65047762f924ecd367c17eabf8f:
+         balance: 0
+         code: '0xEFCAFE01010002020004006000AABBCCDD'
+         nonce: 1
+    - indexes:
+       data: ':label deploying_valid_eof1'
+       gas: !!int -1
+       value: !!int -1
+      network:
+        - 'London'
+      result:
+       a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+         nonce: 1
+         balance: 30000
+       6295ee1b4f6dd65047762f924ecd367c17eabf8f:
+         shouldnotexist: 1
+    - indexes:
+       data: ':label deploying_valid_eof1'
+       gas: !!int -1
+       value: !!int -1
+      network:
+        - 'London+3540'
```

Comment:
@winsvega Weird that it passes CI currently, although it contains this invalid network.

Reply:
CI checks that tests are filled and that the filler hash in the filled test match the hash of Filler file